### PR TITLE
Update release notes (#1597)

### PR DESCRIPTION
**Pull request by kozbo**

### CODE REVIEW

**Comment by david4096** [comment]
Converters were removed.

File: docs/status.rst [hunk 1]
```diff
@@ -20,6 +20,45 @@ Release Notes
...
+
+- Added **support for BigWig files** in a new Continuous Data object #1573
+
+- Add **ability to list and join peer server networks** #1507
+
+- Remove feature_id from ExpressionLevel #1580
+
+- Replaced info fields with rich type Attributes fields #1521
+
+- Replace NCBI taxon ID integer with ontology term #1551
```
Comment (david4096) [inline docs/status.rst]: Say what interfaces this affects.

Comment (kozbo) [inline docs/status.rst]: OK, I added more details. How does that look?

File: docs/status.rst [hunk 2]
```diff
@@ -47,7 +47,7 @@ Features:
  - AnnouncePeerResponse (new)
  - GetInfoResponse (new)
 
-- Remove feature_id from ExpressionLevel #1580  Impacts
+- Remove feature_id from ExpressionLevel and add ability to search by the Name ffield #1580  Impacts
```
Comment (david4096) [inline docs/status.rst]: typo

File: docs/status.rst [hunk 3]
```diff
@@ -20,6 +20,82 @@ Release Notes
...
+This an alpha pre-release that contains some major updates. The most important 
+changes are highlighted in **bold** below. We have also made some updates to 
+the documentation.
+
+Features:
+
+- **Rename package from “ga4gh” to “ga4gh-server”** #1582, #1583
+
+- Added **support for BigWig files** in a new Continuous Data object #1573  New endpoints and message types include:
+ - `POST /continuoussets/search`
```
Comment (david4096) [inline docs/status.rst]: Same thing as on schemas, these backticks aren't doing what you think they might be.

**Pull request merged**